### ISSUE TITLE
OSPRay isosurfaces do not seem to work

### ISSUE THREAD

**Issue by clyne** [NCAR/VAPOR#2450]
I'm unable to render an isosurface using OSPRay with any data I've tested. The application hangs indefinitely much like it does with the volume renderer for some data: #2449. Unlike the volume renderer there is no way to control the isosurface sampling rate.

To reproduce:
1. Load Duku.nc
2. Enable the isosurface renderer with OSPRay selected as the rendering algorithm

**Comment by clyne** [comment]
Correction. The ospray isosurface renderer seems to work fine with regular gridded data, but not a all for curvilinear or unstructured dat.

**Comment by clyne** [comment]
Note: isosurfaces are currently disabled until Intel fixes their bug.

**Comment by clyne** [comment]
We should retest this with the latest version of OSPRay

**Comment by StasJ** [comment]
I have done some experimenting with OSPRay 2.8 and it seems to be working.

**Comment by StasJ** [comment]
> It's working, but the image isn't terribly satisfying; lots of holes suggesting the sampling is inadequate. See attached image and session file. Can this be fixed? Reopening.

No, unfortunately OSPRay does not provide any controls over the isosurface renderer. The volume renderer controls are nops.

**Comment by clyne** [comment]
We should revisit with the next release of OSPRay (2.9?)

**Comment by sgpearse** [comment]
@sgpearse will reverify

**Comment by sgpearse** [comment]
We currently distribute ospray 2.11.  Need to retest.

**Comment by sgpearse** [comment]
Might be fixed with upgrading to Anari?
